content/docs/tasks/traffic-management/egress/index.md

```diff
@@ -264,6 +264,102 @@ $ export SOURCE_POD=$(kubectl get pod -l app=sleep -o jsonpath={.items..metadata
 $ kubectl exec -it $SOURCE_POD -c sleep curl http://httpbin.org/headers
 {{< /text >}}
 
+## Install Istio with access to all external services by default
+
+An alternative to calling external services directly is to instruct the Istio proxy to pass through the calls to all the
+external services. This option allows you to start evaluating Istio quickly, without controlling access to external
+services, and decide to [configure access to external services](#configuring-istio-external-services) later.
+
+Istio has an installation option that allows access to all external services for which no HTTP service exists inside
+the mesh and no HTTP/TCP `ServiceEntry` is defined. For example, if your mesh does not have an HTTP service on port 443,
```

Comment:
I think this sentence is confusing.  what do you mean by no HTTP service exists inside the mesh?  Does it matter the service name?  Or only the port number 443 matters?   It seems uncommon to have http on port 443, should we even mention it here?  If I'm a user, i'd be caring more about the exact install option here.

Reply:
Let's take some other port, let's say 8000. If there will be an HTTP service deployed inside the mesh on the port 8000, the access to external services on the port 8000 will be blocked. The service name does not matter. This is valid for any port - once you deploy an HTTP service in the mesh on some port, the egress traffic to that port will be blocked.

Reply:
Alternatively, once you define anything on port X, except for non-HTTP services inside the mesh, Istio starts blocking external traffic on port X.

Reply:
ok understood.  I think the confusing part is port 443 for an http service.   I would not mention it until a troubleshooting section.

Reply:
I think we have to state clearly in the beginning which ports will be blocked and which ones will be not blocked, this is pretty important for the users. Otherwise, they can start working with Istio, access external services on some port, later define some service on that port and will be surprised why all other external services on that port are blocked.

Reply:
@linsun I can replace the port 443 with some other port in the description, like 8000, or with `port X`.

Reply:
Addressed in https://github.com/istio/istio.io/pull/3222/commits/c841ed4f438123acf09e3fec3c08c4e5e30578e5. 

Used port 8000 instead of 443, to make it less confusing. Also added "inside the mesh" to "if you later create an HTTP service". It is important to mind what creates a conflict on some port:

1. An HTTP service inside the mesh
2. Any service entry for any service. The service entry is defined, naturally, inside the mesh. The  service that the service entry is defined for, can reside inside or outside the mesh.